.github/actions/bpftrace/scripts/check-encryption-leaks.bt

```diff
@@ -153,7 +163,7 @@ kprobe:br_forward
         $tcph = (struct tcphdr*)$udph;
 
         if (!$tcph->rst) {
-          printf("[%s] %s:%d -> %s:%d (proto: %d, TCP flags: %c%c%c%c, encap: %d, ifindex: %d, netns: %x, override: %d)\n",
+          printf("[%s] %s:%d -> %s:%d (proto: %d, TCP flags: %c%c%c%c, encap: %d(skb: %d), ifindex: %d, netns: %x, override: %d)\n",
```

Comment:
Nit: Missing space, no?

Reply:
I'm thinking whether to add the space or not :thinking: 
Considering upcoming PRs, I'm afraid the resulting output will be longer than 1-line...
But maybe as far as it concerns this PR, the space would be good.

Reply:
If we want to make it shorter than one line, we can split it on two lines with proper indentation.

Reply:
For now I'll leave it there (adding the space).
In subsequent PRs, if I'm still willing to add additional trace info which would lead to having a very long line I'll split. Unfortunately, the only way to do that in bpftrace would be either:
1. a very long print statement: not possible to split string in multiple lines -- pessimal readability
2. multiple print statements: would avoid if possible, might become hard to read in logs if we have 2/3 prints per flow.